deps/target-deps.json

```diff
@@ -6,19 +6,19 @@
                 "name" : "slang-llvm",
                 "baseUrl" : "https://github.com/shader-slang/slang-llvm/releases/download/v13.x-42/",
                 "optional" : true,
-                "packages" :
+                "packages" : 
```

Comment:
A lot of superfluous changes in this file. Just reset this file?

Reply:
This file was already reset in the latest commit.